src/gpl/src/nesterovBase.cpp

```diff
@@ -1922,14 +1922,17 @@ void NesterovBase::updateAreas()
                           / static_cast<float>(whiteSpaceArea_);
 
   if (totalFillerArea_ < 0) {
-    log_->error(GPL,
-                303,
-                "Use a higher -density or "
-                "re-floorplan with a larger core area.\n"
-                "Given target density: {:.2f}\n"
-                "Suggested target density: {:.2f}",
-                targetDensity_,
-                uniformTargetDensity_);
+    log_->warn(
+        GPL,
+        303,
+        "Not enough movable area, negative totalFillerArea_ in updateAreas(). "
```

Comment:
I would avoid using internal variable names in warning messages. They are meaningless for the user and can cause confusion.